### PR TITLE
[Feature #68059]: Quản lý đơn hàng

### PR DESCRIPTION
**Pull request by BuiMinhDung711**
## Related Tickets:
- [#68063](https://edu-redmine.sun-asterisk.vn/issues/68063)
- [#68327](https://edu-redmine.sun-asterisk.vn/issues/68327)
## WHAT this PR does?

- Create a order view with filter

## Notes Impacted Areas:
- N/A.

## Attachments:
- image

### CODE REVIEW

**Comment by BuiMinhDung711** [comment]
ready

File: app/Http/Controllers/OrderController.php [hunk 1]
```diff
@@ -89,4 +94,21 @@ public function placeOrder(PlaceOrderRequest $request, $totalPayment)
...
+     */
+    public function showAdmin(Bill $bill)
+    {
+        $bill = Bill::with([
+            'user',
+            'billProducts',
+            'billProducts.productInStock',
+            'billProducts.productInStock.product',
+        ])->find($bill->id);
+        return view('order.show', ['bill' => $bill]);
```
Comment (ghost) [inline app/Http/Controllers/OrderController.php]:
```suggestion
        ])->find($bill->id);

        return view('order.show', ['bill' => $bill]);
```

**Comment by BuiMinhDung711** [comment]
ready

**Comment by BuiMinhDung711** [comment]
ready

**Pull request merged**